### PR TITLE
new file:   packages/linwrap/linwrap.9.1.4/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
some minor bugs corrected + some minor cosmetic changes

### CODE REVIEW

**Comment by kodwx** [comment]
Consider adding `"ocaml" {>= "4.12.0"}` as a dependency.

File: packages/linwrap/linwrap.9.1.4/opam [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/linwrap"
+bug-reports: "https://github.com/UnixJunkie/linwrap/issues"
+dev-repo: "git+https://github.com/UnixJunkie/linwrap.git"
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: ["cp" "bin/ecfp6.py" "%{bin}%/linwrap_ecfp6.py"]
+depends: [
+  "base-unix"
+  "batteries" {>= "3.0.0"}
```
Comment (mseri) [inline packages/linwrap/linwrap.9.1.4/opam:12]:
```suggestion
  "batteries" {>= "3.3.0"}
```
Uses `BatList.unfold_exn` which was introduced in batteries 3.3.0.

**Comment by mseri** [comment]
Looks fine. Opam is failing to produce a resolution of ocaml < 4.07, but it may well be that with the right set of packages the library compiles so I am not going to add any extra bound fpr now

**Comment by UnixJunkie** [comment]
thanks, I merged upstream changes to the opam file

**Pull request merged**